components/brave_ads/core/internal/account/utility/redeem_confirmation/reward/redeem_reward_confirmation_unittest.cc

```diff
@@ -31,6 +31,12 @@ namespace brave_ads {
 
 class BraveAdsRedeemRewardConfirmationTest : public UnitTestBase {
  protected:
+  void RunPendingFetchPaymentTokenTask() {
+    FastForwardClockToNextPendingTask();
+  }
+
+  bool HasPendingFetchPaymentTokenTask() const { return HasPendingTasks(); }
```

Comment:
nit: not sure we need this because `HasPendingTasks();` is clear as to what is happening.

Reply:
We may want to check we only have a single pending task using PendingTaskCount

Reply:
> nit: not sure we need this because `HasPendingTasks();` is clear as to what is happening.

Fixed